### DISCUSSION TITLE
Rename argument "none" to a better name since it is very easy to missread it as None

### DISCUSSION THREAD

**Discussion by Harding-Stardust** [arizvisa/ida-minsc#181] [Q&A] [answered]
https://github.com/arizvisa/ida-minsc/blob/af07f167a0992aed3f3a2a1670593880baa57820/base/database.py#L1791

**Comment by arizvisa** [comment]
Actually, that parameter is **actually** intended to be read as `None`. If you pass `None` as that parameter, it will remove the comment. In all cases, that implementation will \_only\_ have `None` as the value for the parameter.

**Comment by Harding-Stardust** [comment]
I am having some problems to wrap my head around this. This is to allow the user to send in None instead of an empty string?

**Reply by arizvisa** [reply, marked as answer]
Yep. Anytime you give `None` as a function parameter, it will **remove** the thing as opposed to just setting it to empty. This is used in things like tags, colors, types, variable names, etc.

With tags, you can assign an empty string to them. With colors, assigning the value 0 makes it black, whereas `None` removes the color. Variable names when set to `None` make it use the default variable name.
